apis/fluentbit/v1alpha2/plugins/output/splunk_types.go

```diff
@@ -0,0 +1,137 @@
+	SplunkToken string `json:"splunk_token,omitempty"`
+	//Buffer size used to receive Splunk HTTP responses: Default `2M`
+	// +kubebuilder:validation:Pattern:="^\\d+(k|K|KB|kb|m|M|MB|mb|g|G|GB|gb)?$"
+	BufferSize string `json:"bufferSize,omitempty"`
+	// Set payload compression mechanism. The only available option is gzip.
+	Compress string `json:"compress,omitempty"`
+	// Specify X-Splunk-Request-Channel Header for the HTTP Event Collector interface.
+	Channel string `json:"channel,omitempty"`
+
+	// Optional username credential for access
+	HTTPUser *plugins.Secret `json:"httpUser,omitempty"`
+	// Password for user defined in HTTP_User
+	HTTPPasswd *plugins.Secret `json:"httpPassword,omitempty"`
+	// If the HTTP server response code is 400 (bad request) and this flag is enabled, it will print the full HTTP request
+	// and response to the stdout interface. This feature is available for debugging purposes.
+	HTTPDebugBadRequest bool `json:"httpDebugBadRequest,omitempty"`
+	// When enabled, the record keys and values are set in the top level of the map instead of under the event key. Refer to
+	// the Sending Raw Events section from the docs for more details to make this option work properly.
+	SplunkSendRaw bool `json:"splunkSendRaw,omitempty"`
+	//Specify the key name that will be used to send a single value as part of the record.
+	EventKey string `json:"eventKey,omitempty"`
+	//Specify the key name that contains the host value. This option allows a record accessors pattern.
+	EventHost string `json:"eventHost,omitempty"`
+	//Set the source value to assign to the event data.
+	EventSource string `json:"eventSource,omitempty"`
+	//Set the sourcetype value to assign to the event data.
+	EventSourcetype string `json:"eventSourcetype,omitempty"`
+	// Set a record key that will populate 'sourcetype'. If the key is found, it will have precedence
+	// over the value set in event_sourcetype.
+	EventSourcetypeKey string `json:"eventSourcetypeKey,omitempty"`
+	// The name of the index by which the event data is to be indexed.
+	EventIndex string `json:"eventIndex,omitempty"`
+	// Set a record key that will populate the index field. If the key is found, it will have precedence
+	// over the value set in event_index.
+	EventIndexKey string `json:"eventIndexKey,omitempty"`
+	//Set event fields for the record. This option can be set multiple times and the format is key_name
+	// record_accessor_pattern.
+	EventField string `json:"eventField,omitempty"`
+
+	// Enables dedicated thread(s) for this output. Default value `2` is set since version 1.8.13. For previous versions is 0.
+	Workers      *int32 `json:"Workers,omitempty"`
+	*plugins.TLS `json:"tls,omitempty"`
+}
+
+// Name implement Section() method
+func (_ *Splunk) Name() string {
+	return "splunk"
+}
+
+// Params implement Section() method
+func (o *Splunk) Params(sl plugins.SecretLoader) (*params.KVs, error) {
+	kvs := params.NewKVs()
+	if o.Host != "" {
+		kvs.Insert("Host", o.Host)
```

Comment:
Uppercase is not recommended for code consistency.

kvs.Insert("Host", o.Host) => kvs.Insert("host", o.Host)

Others also need to be modified.